test/cli/template-create/complex/out/.keboola/repository.json

```diff
@@ -9,6 +9,7 @@
       "id": "my-template",
       "name": "My Template",
       "description": "Full workflow to ...",
+      "requirements": {},
```

Comment:
I prefer empty values, not `omitempty`.

It is result of the `template-create` command, ... and the template author don't know which keys can be inside the `requirements` object.